frontend/src/actions/index.js

```diff
@@ -142,7 +142,21 @@ export const postPost = (post) => dispatch => (
     .then(function(response) {
         return response.json()
     })
-);
+};
+
+// export const postPost = (post) => dispatch => (
+//     fetch("http://localhost:3001/posts", {
+//         headers: {'Authorization': 'whatever-you-want'},
+//         method: 'POST',
+//         data: {
+//             ...post
+//         }
+//     })
+//     .then(function(response) {
+//         console.log(response)
+//         return response.json()
+//     })
+// );
```

Comment:
so when you add a new post you dont need to dispatch the fetch to your server, because you already do that with the add post action. so i just created a new function in the edit post component.